src/TableContent.js

```diff
@@ -47,10 +47,18 @@ const TableContent = ({tableHeader, searchedColumnIndex, searchQuery, orderedCol
 
                 {[
                   tableHeader.map((header, index) => {
-                    const cellItem = Array.isArray(data[header.dataParam]) ?
-                      <ul key={`cell${index}`}>{data[header.dataParam].map(element => <li key={element}>{element}</li>)}</ul> :
-                      data[header.dataParam]
-                    return <Table.Cell key={`${cellItem}`} flexBasis={header.width} flexShrink={100} flexGrow={100}>
+                    let cellItem
```

Comment:
Whenever you have an assignment that depnds on a condition, prefer this construct:
```
const variable = condition ? value-when-condition-is-true : value-when-condition-is-false
```

In this way you maintain the safet of `const` and clearly transmits the intent to other team developers that the value won’t change, making the cognitive burden lighter.